specification/riskiq/Easm/models.tsp

```diff
@@ -1671,9 +2034,121 @@ model MarkParameter {
   mark?: string;
 }
 
+@doc("Specify the response type.")
+@Versioning.added(Versions.v2024_03_01_preview)
+model ResponseTypeParameter {
+  @doc("Specify the response type. The possible values are: ID, STANDARD, FULL, REDUCED")
+  @query("responseType")
+  responseType?: AssetResponseType;
+}
+
+@doc("The prameter of responseIncludes")
+@Versioning.added(Versions.v2024_03_01_preview)
+model ResponseIncludesParameter {
+  @query({
+    format: "csv",
+  })
+  @doc("The properties to include in the response.")
+  responseIncludes?: string[];
+}
+
+@doc("The prameter of responseIncludes")
+@Versioning.added(Versions.v2024_03_01_preview)
+model RecentOnlyParameter {
+  @query("recentOnly")
+  @doc("If it's recent only.")
```

Comment:
How "recent" is recent?

Reply:
it varies. this recent is a internal term we use, definition is below:

<html>
<body>
<!--StartFragment--><span><span class="ui-provider a b c d e f g h i j k l m n o p q r s t u v w x y z ab ac ae af ag ah ai aj ak" dir="ltr"><h2 style="margin-bottom: 16px; margin-top: 16px;"><strong>RECENCY</strong></h2><p style="margin-bottom: 16px; margin-top: 8px;">Currently GI API and UI search both default to searching against "recent" data. There is a special dropdown in the UI and a search history API that will search against historical data.</p><p style="margin-bottom: 16px; margin-top: 8px;">We define recency based on the datasource from which the various asset fields come from and whether the last seen on those fields makes the cutoff.